### PR TITLE
Fonts nav restyling

### PR DESCRIPTION
**Pull request by OBackh**
Adjust some styling at:
- Navbar (white 1-Pixel border and shadow, actual status of navigation/page)
- Headlines (colors)
- Styled table (Workout >show more<)

### CODE REVIEW

File: components/Layout.js [hunk 1]
```diff
@@ -31,14 +46,16 @@ const Nav = styled.nav`
   justify-content: center;
 `;
 
-const NavLink = styled(Link)`
-  color: #ffffff;
+const NavLink = styled(({ isActive, ...props }) => <Link {...props} />)`
```
Comment (NikRoe) [inline components/Layout.js:49]: This is a fancy approach!

What you see in the docs from Styled Components is the prefixing of the props passed to a styled component with "$" to filter them. But your solution works just as well 👍

File: components/Layout.js [hunk 2]
```diff
@@ -49,7 +66,7 @@ const NavLink = styled(Link)`
   &:active {
     background-color: #ffffff;
     color: #3498db;
-    font-weight: bold;
+    _font-weight: bold;
```
Comment (NikRoe) [inline components/Layout.js:69]: If you don't need a property, I would just remove it.

File: components/Layout.js [hunk 3]
```diff
@@ -59,6 +76,8 @@ const MainContent = styled.main`
   padding: 1rem;
   padding-top: 0;
   margin-bottom: 65px;
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  _background-color: #efefef;
```
Comment (NikRoe) [inline components/Layout.js:80]: If you don't need a property, I would just remove it.

File: components/Workout/Workout.js [hunk 4]
```diff
@@ -155,25 +161,49 @@ const ExerciseList = styled.ul`
...
+  text-align: left;
+  font-size: 0.9rem;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
 `;
 
-const ExerciseName = styled.h3`
+const ExerciseName = styled.span`
+  _flex-grow: 1;
```
Comment (NikRoe) [inline components/Workout/Workout.js:185]: If you don't need a property, I would just remove it.

File: components/Workout/Workout.js [hunk 5]
```diff
@@ -155,25 +161,49 @@ const ExerciseList = styled.ul`
...
   margin: 0;
-  color: #333;
+  font-weight: 500;
+  color: #222;
+  margin-right: 0.5rem;
 `;
 
-const ExerciseDetails = styled.p`
+const ExerciseDetails = styled.span`
+  _flex-grow: 0;
```
Comment (NikRoe) [inline components/Workout/Workout.js:194]: If you don't need a property, I would just remove it.

**Review by NikRoe — APPROVED** [review]
Hi there,

nice work 🙌 

I left some comments for the CSS properties that don't get applied. If you don't need them, just remove them.

After having done so, you can continue to QA without having to repost the PR.

Keep it up 🙌

File: components/Layout.js (hunk 1 above)
Comment (OBackh) [reply components/Layout.js:49]: Thank you Niklas, this is the solution from gpt.

File: components/Layout.js (hunk 2 above)
Comment (OBackh) [reply components/Layout.js:69]: Just removed! :)

File: components/Workout/Workout.js (hunk 4 above)
Comment (OBackh) [reply components/Workout/Workout.js:185]: Just removed! :)

File: components/Layout.js (hunk 3 above)
Comment (OBackh) [reply components/Layout.js:80]: Just removed! :)

File: components/Workout/Workout.js (hunk 5 above)
Comment (OBackh) [reply components/Workout/Workout.js:194]: Just removed! :)

**Pull request merged**